apps/addons/tests/test_commands.py

```diff
@@ -211,3 +213,22 @@ def test_approve_addons_get_review_type(use_case):
     """
     addon, file1, _, review_type = use_case
     assert approve_addons.get_review_type(file1) == review_type
+
+
+# fix_let_scope_bustage.
```

Comment:
I thought it was helping separate the previous tests from the ones related to the "fix_let_scope_bustage" code. Any recommendation? Should I just remove it?